autotools.sh

```diff
@@ -27,81 +28,63 @@ esac
 echo "Building ALICE autotools. To avoid this install autoconf, automake, autopoint, texinfo, pkg-config."
 
 # Restore original timestamps to avoid reconf (Git does not preserve them)
-pushd $SOURCEDIR
+pushd "$SOURCEDIR" || exit 1
   ./missing-timestamps.sh --apply
-popd
+popd || exit 1
 
-rsync -a --delete --exclude '**/.git' $SOURCEDIR/ .
+rsync -a --delete --exclude '**/.git' "$SOURCEDIR"/ .
 
 # Use our auto* tools as we build them
 export PATH=$INSTALLROOT/bin:$PATH
 export LD_LIBRARY_PATH=$INSTALLROOT/lib:$LD_LIBRARY_PATH
 
 # help2man
 if pushd help2man*; then
-  ./configure --disable-dependency-tracking --prefix $INSTALLROOT
+  ./configure --disable-dependency-tracking --prefix "$INSTALLROOT"
   make ${JOBS+-j $JOBS}
   make install
   hash -r
-  popd
+  popd || exit 1
```

Comment:
What's the exit 1 for?

Reply:
Oh my bad, I had shellcheck misconfigured on my local and it wasn't detecting the `bash -e` in the shebang. Will remove it